lang/cs/Org.Apache.REEF.Driver/Bridge/Events/FailedTask.cs

```diff
@@ -33,7 +32,18 @@ internal sealed class FailedTask : IFailedTask
         
         public FailedTask(IFailedTaskClr2Java failedTaskClr2Java)
         {
-            Parse(failedTaskClr2Java);
+            var serializedInfo = failedTaskClr2Java.GetFailedTaskSerializedAvro();
+            var avroFailedTask = AvroJsonSerializer<AvroFailedTask>.FromBytes(serializedInfo);
+
+            Id = avroFailedTask.identifier;
+            Data = Optional<byte[]>.OfNullable(avroFailedTask.data);
+            Message = avroFailedTask.message ?? "No message in Failed Task.";
+
+            // TODO[JIRA REEF-1258]: Fill this in with avroFailedTask.cause.
+            Cause = Optional<Exception>.Empty();
```

Comment:
Are we able to get exception type from the Cause once REEF-1258 is done? We need typeof(Exception) at driver side. If not, can we add this property in AvroFailedTask and FailedTask?